v3/how-to-guides/07-parachains/a-connect-relay/index.mdx

```diff
@@ -5,8 +5,7 @@ version: polkadot-0.9.10
 section: how to guides
 category: parachains
 difficulty: 1
-keywords:
-  [collator, parachain, parathread, upgrade, cumulus, storage, migration, paraid, register, launch]
+keywords: collator, parachain, parathread, upgrade, cumulus, storage, migration, paraid, register, launch
```

Comment:
With _many) keywords I would want _all_ of them to be an array, as it can be much easier to read and sort (with prettier #482 )

Reply:
@NukeManDan 
For sake of time and simplicity, I just reverted the three markdown pages that had arrays to strings for now.  Mainly to get rid of the CI error in gatsby.  
This is also not configured in search at the moment, so strings are acceptable vs arrays.  If we want to switch it all to arrays, we should do it in a one go and update our search to account for it.  Or else it's just going to output null and not count keywords towards the final score. 

Perhaps we can quickly discuss during the next stand-in.

Reply:
Created a card on our project board to capture this: https://github.com/substrate-developer-hub/substrate-docs/projects/2#card-72531517